build/common.props

```diff
@@ -35,6 +35,12 @@
     <LangVersion>11</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageVersion>7.1.0</PackageVersion>
+    <EnablePackageValidation>true</EnablePackageValidation>
+    <PackageValidationBaselineVersion>7.0.2</PackageValidationBaselineVersion>
```

Comment:
Shouldn't the baseline be 7.0.0? Otherwise how to find breaking changes compared to a major version